drivers/storage/portworx/component/csi.go

```diff
@@ -686,6 +699,54 @@ func getCSIDeploymentSpec(
 		)
 	}
 
+	if csiConfig.IncludeHealthMonitorController && healthMonitorControllerImage != "" {
+		deployment.Spec.Template.Spec.Containers = append(
+			deployment.Spec.Template.Spec.Containers,
+			v1.Container{
+				Name:            csiHealthMonitorControllerContainerName,
+				Image:           healthMonitorControllerImage,
+				ImagePullPolicy: imagePullPolicy,
+				Args: []string{
+					"--v=3",
+					"--csi-address=$(ADDRESS)",
+					"--leader-election",
```

Comment:
In the other components the flag is `--leader-election=true`. Does passing just `--leader-election` work in the health monitor controller?

Reply:
Yes, that's correct - `--leader-election` is the new standardized way csi sidecars. New sidecar major versions use this flag